### ISSUE TITLE
Generic error wrapper to return Meteor.Error

### ISSUE THREAD

**Issue by dburles** [jazzband/django-ddp#4]
For example calling an unknown method returns the following:
`a["{\"msg\": \"error\", \"error\": \"Unknown method: foo\"}"]`

Meteor won't pass errors like the above through to the `error` argument of the callback (example below).

```
Django.call('foo', function(error, response) {
  if (error)
    console.log(error);
});
```

**Comment by dburles** [comment]
@tysonclugg looks like this is done! ? :+1:

**Comment by tysonclugg** [comment]
Thanks @dburles - indeed this was fixed in d762fa45c498979fdb704567898f3bb028415861. :smiley:
